test/clj/puppetlabs/trapperkeeper/services/webserver/jetty10_config_test.clj

```diff
@@ -69,6 +69,7 @@
       (update-in [:https :protocols] (fnil identity default-protocols))
       (update-in [:https :client-auth] (fnil identity default-client-auth))
       (update-in [:https :allow-renegotiation] (fnil identity default-allow-renegotiation))
+      (update-in [:https :sni-required] (fnil identity sni-required))
```

Comment:
Looks like `sni-required` is undefined in this file.

Reply:
Ah refer all, n/m

Reply:
covered by the `puppetlabs.trapperkeeper.services.webserver.jetty10-config :refer :all` require. (I prefer the explicit `:as` form in general but didn't want to refactor for this)